src/ToastNotification/ToastNotification.js

```diff
@@ -0,0 +1,65 @@
+import ClassNames from 'classnames'
+import React from 'react'
+import PropTypes from 'prop-types'
+
+/**
+ * ToastNotification Component for Patternfly React
+ */
+const ToastNotification = ({
+  notificationClass,
+  children,
+  onDismiss,
+  type
+}) => {
+  const notificationClasses = ClassNames(
+    {
+      alert: true,
+      'toast-pf': true,
+      'alert-danger': type === 'danger' || type === 'error',
+      'alert-warning': type === 'warning',
+      'alert-success': type === 'success',
+      'alert-info': type === 'info',
+      'alert-dismissable': onDismiss
+    },
+    notificationClass
+  )
+  const iconClass = ClassNames({
+    pficon: true,
+    'pficon-error-circle-o': type === 'danger' || type === 'error',
+    'pficon-warning-triangle-o': type === 'warning',
+    'pficon-ok': type === 'success',
+    'pficon-info': type === 'info'
+  })
+
+  return (
+    <div className={notificationClasses}>
+      {onDismiss &&
+        <button
+          type="button"
+          className="close"
+          aria-hidden="true"
+          onClick={onDismiss}
+        >
+          <span className="pficon pficon-close" />
```

Comment:
i like this idea (have seen it used in other frameworks), but I am only seeing `Glyphicons` in React Bootstrap. Can you propose one you'd like me to try or maybe file a subsequent PR after this? Also not sure as to the syntax we'd like...